Extension/src/Debugger/extension.ts

```diff
@@ -91,12 +91,15 @@ export function initialize(context: vscode.ExtensionContext): void {
                         await util.ensureBuildTaskExists(selection.configuration.preLaunchTask);
                         Telemetry.logDebuggerEvent("buildAndDebug", { "success": "false" });
                     } catch (e) {
+                        if (e === util.failedToParseTasksJson) {
```

Comment:
if you make the e an `Error` instead of a raw string, then this needs to be updated.